azure-pipelines.yml

```diff
@@ -54,37 +69,36 @@ jobs:
     msbuildExtraArguments: '/nowarn:VSX1000 /p:CreateAllAndroidTargets=true /bl:$(Build.ArtifactStagingDirectory)\win.binlog'
     buildConfiguration: $(DefaultBuildConfiguration)
     buildPlatform: $(DefaultBuildPlatform)
```

Comment:
I think most of the code below this line doesn't needs to be changed does it?

Reply:
nop ,